.github/scripts/ep_hooks.py

```
@@ -55,6 +55,13 @@ def _sanitize_text(text, max_len=500):
                        '[link removed]', text)
         return text.strip()[:max_len]
 
+    @staticmethod
+    def _write_step_summary(ticket_key, cost_summary):
```

Ruff (0.15.20)

[warning] Missing return type annotation for staticmethod `_write_step_summary`

Add return type annotation: `None`

(ANN205)